writings/updated/08-chapter-7.adoc

```diff
@@ -1351,9 +1351,6 @@ who through death obtained this resurrection, and eternal holy life.
 
 --
 
-[.asterism]
-'''
```

Comment:
So, good instinct here to remove the asterism because it's right after an embedded document. 👍

However... this is a case where I think we'll keep it, because the very next thing it says is that he died, it sort of feels appropriate to separate his dying testimony from the stuff that follows.

So, since this is the only change here, I'll just close this PR.  But again, good eye! Usually this would be a delete.